code/modules/power/apc/apc_attack.dm

```diff
@@ -139,12 +139,15 @@
 		if(!(machine_stat & BROKEN || opened == APC_COVER_REMOVED || atom_integrity < max_integrity)) // There is nothing to repair
 			balloon_alert(user, "no reason for repairs!")
 			return
-		if(!(machine_stat & BROKEN) && opened == APC_COVER_REMOVED) // Cover is the only thing broken, we do not need to remove elctronicks to replace cover
+		if((machine_stat & BROKEN) && opened == APC_COVER_REMOVED && APC_ELECTRONICS_SECURED && terminal) // Cover is the only thing broken, we do not need to remove elctronicks to replace cover
```

Comment:
What is ``APC_ELECTRONICS_SECURED`` doing? You're not checking it to anything (like ``has_electronics`` which is where it's used), just that it's true.

Reply:
now that i think about it, the terminal should cover it anyway since you cant have a terminal without electronics first